js/validator.js

```diff
@@ -9,12 +9,59 @@ export const Validator = {
             errorType: 'required',
         },
         isNumber: {
-            validate: (value) => !isNaN(value) && Number.isInteger(+value),
+            validate: (value) => !Number.isNaN(value) && Number.isInteger(+value),
             message: 'The field should has a integer',
             errorType: 'number',
         },
-        maxLength() {
-            // need to add maxLength option
+        maxLength(value) {
+            const validate = (valueInput) => {
+                return valueInput.length < value;
+            };
+
+            const message = 'You have exceeded the maximum value';
+            const errorType = 'TypeError';
```

Comment:
errorType must be specific for each validator's method. For example: for isNumber - should be 'number', for checkEmail - should be 'email', for password - 'password', and 'password_match'